PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -439,7 +439,8 @@ def _add_jetsPuppi(process):
     
         process.patJetsPuppi.jetChargeSource = cms.InputTag("patJetPuppiCharge")
     
-        process.selectedPatJetsPuppi.cut = cms.string("pt > 15")
+        process.selectedPatJetsPuppi.cut = cms.string("pt > 10")
+        (_run2_miniAOD_ANY | pA_2016).toModify(process.selectedPatJetsPuppi, cut = "pt > 15")
```

Comment:
what's the reason to exclude these particular eras?

Reply:
if the intent is to change it for Run3, would it perhaps be better to change the value **only** for a Run3 modifier, rather than avoid changing the value for some Run2 modifiers?

Reply:
My apologies @jpata. I misspoke. It is for beginning from Run 3 and onwards. Not only just for Run 3.

Reply:
talking more with Slava, we think a modifier should not be applied here at all. There is a general rule to keep the configuration for all eras as similar as possible, and explicitly, there is no commitment to being able to maintain old cuts with new CMSSW versions for old data eras.

Is there some very strong physics or technical reason why you want to exclude Run2 from this change?

Reply:
No, there is no strong physics case or technical reasons do this. I suppose this stems from me personally not knowing what are the rules/guidelines on making configuration changes and what to keep constant for old eras (Apologies again for the confusion). If it is your recommendation to not apply a modifier, I would be happy to remove it.

Reply:
thanks, please remove it then.
(btw note that `_run2_miniAOD_ANY` would have changed it for reMINI, but not for a reRECO of Run2 AFAIK, leading to a potential inconsistency down the line. yet more reason to *not* apply modifiers unless strongly motivated.)